go.sum

```diff
@@ -117,6 +117,8 @@ github.com/apache/dubbo-go-hessian2 v1.8.2 h1:CQq2Mmlrk6Fqmudwl9Dqps8drTrBFnmXRl
 github.com/apache/dubbo-go-hessian2 v1.8.2/go.mod h1:xQUjE7F8PX49nm80kChFvepA/AvqAZ0oh/UaB6+6pBE=
```

Comment:
these history can be deleted

Reply:
done, by the way, why we need delete these history ?